### PR TITLE
Update Remove-MpPreference.md

### PR DESCRIPTION
**Pull request by msbemba**
Added AttackSurfaceReductionOnlyExclusions to the syntax, added an example, and Parameter description.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1563

**Comment by e0i** [comment]
cc @dariomws @get-itips @tseward

File: docset/windows/defender/Remove-MpPreference.md [hunk 1]
```diff
@@ -69,6 +77,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+
+### -AttackSurfaceReductionOnlyExclusions
+Exclude files and paths from Attack Surface Reduction (ASR) rules. Specify the folders or files and resources that should be excluded from ASR rules. Enter a folder path or a fully qualified resource name. For example, ""C:\Windows"" will exclude all files in that directory. ""C:\Windows\App.exe"" will exclude only that specific file in that specific folder.
+
+For more information about excluding files and folders from [ASR rules](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/enable-attack-surface-reduction#exclude-files-and-folders-from-asr-rules)
```
Comment (dariomws) [inline docset/windows/defender/Remove-MpPreference.md:84]:
```suggestion
For more information about excluding files and folders from [ASR rules](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/enable-attack-surface-reduction#exclude-files-and-folders-from-asr-rules).
```

File: docset/windows/defender/Remove-MpPreference.md [hunk 2]
```diff
@@ -45,6 +46,13 @@ PS C:\> Remove-MpPreference -ExclusionPath "C:\Temp"
 
 This command removes the folder C:\Temp from the exclusion list.
 
+### Example 2: 
+```
+PS C:\> Remove-MpPreference -AttackSurfaceReductionOnlyExclusions "C:\Windows\App.exe"
+```
+
+This command will exclude only that specific file app.exe in that specific folder
```
Comment (dariomws) [inline docset/windows/defender/Remove-MpPreference.md:54]:
```suggestion
This command will exclude only that specific file app.exe in that specific folder.
```

File: docset/windows/defender/Remove-MpPreference.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Removes exclusions or default actions.
 ```
 Remove-MpPreference [-ExclusionPath <String[]>] [-ExclusionExtension <String[]>] [-ExclusionProcess <String[]>]
  [-ThreatIDDefaultAction_Ids <Int64[]>] [-UnknownThreatDefaultAction] [-LowThreatDefaultAction]
+ [-AttackSurfaceReductionOnlyExclusions]
```
Comment (dariomws) [inline docset/windows/defender/Remove-MpPreference.md:31]:
```suggestion
 [-AttackSurfaceReductionOnlyExclusions <String>]
```

**Pull request merged**